mpd/base.py

```diff
@@ -239,12 +239,20 @@ def _parse_objects(self, lines, delimiters=[], lookup_delimiter=False):
         obj = {}
         for key, value in self._parse_pairs(lines):
             key = key.lower()
-            if lookup_delimiter and not delimiters:
-                delimiters = [key]
+            if lookup_delimiter and key not in delimiters:
+                delimiters = delimiters + [key]
```

Comment:
This is a bit inelegant but it was necessary to prevent `delimiters` from changing permanently. This is a general problem with mutable default values. If you have a better idea to fix this please let me know.